besu/src/main/java/org/hyperledger/besu/cli/BesuCommand.java

```diff
@@ -1177,6 +1217,20 @@ private JsonRpcConfiguration jsonRpcConfiguration() {
           "Unable to authenticate JSON-RPC HTTP endpoint without a supplied credentials file or authentication public key file");
     }
 
+    if (isRpcHttpEnabled && isRpcHttpTlsEnabled) {
```

Comment:
do we want to show a warning if TLS params are specified, but RPC is disabled? Not necessary fail - but log that some cmdline params are not being used?

Reply:
updated.